src/include/libpmemobj.h

```diff
@@ -940,19 +942,19 @@ pmemobj_tx_free((o).oid)
 	TX_ADD_FIELD(o, field);\
 	D_RW(o)->field = value; })
 
-#define	TX_MEMCPY(dest, src, num) (\
-{\
-	void *d = (dest);\
-	size_t n = (num);\
-	pmemobj_tx_add_range_direct(d, n);\
-	memcpy(d, src, n); })
+static inline void
```

Comment:
POSIX memcpy returns a void * (which matches the dest passed in).  Should we be following this interface?